.gitignore

```diff
@@ -1,3 +1,4 @@
 .Rproj.user
 .Rhistory
 .RData
+.DS_Store
```

Comment:
Add .DS_Store to your project .gitignore or to your global one with `usethis::git_vaccinate()`.